src/Components/HomePage/HomePage.css

```diff
@@ -1,72 +1,55 @@
-.phone {
-    position: absolute;
-    top: 175px;
-    width: 700px;
-    box-shadow: 0 8px 24px rgba(0,0,0,.2);
-}
-
-
-.background2 {
-    width: 750px;
-    margin-top: 60px;
-    background-color: grey;
-    height: 485px;
-    z-index: -1;
-    position: absolute;
+.homeHero {
```

Comment:
`Hero` is just the standard name for a large image section at the top of a web page